### PR TITLE
Add ConsoleYamlSamples to the supported resources

### PR DESCRIPTION
**Pull request by Jamstah**
**Description of the change:**
Allow the inclusion of samples in the bundle manifest

https://github.com/operator-framework/operator-lifecycle-manager/issues/1615

**Motivation for the change:**
Ease of deployment of samples for the console

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [ ] Commit messages sensible and descriptive

### CODE REVIEW

File: pkg/lib/bundle/supported_resources.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ const (
 	PodDisruptionBudgetKind   = "PodDisruptionBudget"
 	PriorityClassKind         = "PriorityClass"
 	VerticalPodAutoscalerKind = "VerticalPodAutoscaler"
+	ConsoleYamlSampleKind     = "ConsoleYamlSample"
```
Comment (njhale) [inline pkg/lib/bundle/supported_resources.go:19]: I think these need to be added to OLM before we allow them to be added to bundles: https://github.com/operator-framework/operator-lifecycle-manager/blob/78bb41b4613cafca7264a3108d1f1fa4b66024c8/pkg/controller/operators/catalog/operator.go#L2080

**Review by njhale — CHANGES_REQUESTED** [review]

File: pkg/lib/bundle/supported_resources.go (hunk 1 above)
Comment (njhale) [reply pkg/lib/bundle/supported_resources.go:19]: Ah, I see there's a PR open for it: https://github.com/operator-framework/operator-lifecycle-manager/pull/1616

**Comment by njhale** [comment]
/hold

awaiting https://github.com/operator-framework/operator-lifecycle-manager/pull/1616

**Comment by exdx** [comment]
e2e aws currently having CI issues. this feels safe to override as its a trivial change. 
/override ci/prow/e2e-aws

**Pull request merged**